pom.xml

```diff
@@ -86,7 +86,7 @@
         <assertj.version>3.7.0</assertj.version>
         <jackson.version>2.6.7</jackson.version>
         <jackson-databind.version>2.6.7</jackson-databind.version>
-        <openml-api.version>0.1.0</openml-api.version>
+        <openml-api.version>0.1.1</openml-api.version>
```

Comment:
build failed because this version is still not on maven central. let's wait for some time...

Reply:
build is green